index.html

```diff
@@ -222,7 +221,9 @@ <h2 class="title2">Multi-Post Stories</h2>
   
       <button class="button2" type="button">See project</button>
   </div>
```

Comment:
- [ ] It is required to stick to the figma design as much as possible. This includes simulating every layout on the figma design.  Your design must look exactly like that of figma design. Kindly check the `See Project` button on your design as it deos not look like how it is on the figma design. The text should be on a line. You might need to increase the padding of the button to fix ths issue. Ensure you correct this to every other projects

###  Your Design
![your-see-project-btn-1](https://user-images.githubusercontent.com/26597432/221514897-8706922b-9850-4e68-ba12-ccf1644810e7.png)

### Figma Design
![temp-see-project-btn](https://user-images.githubusercontent.com/26597432/221514944-f9fee8f5-9329-4292-9588-0affd4841166.png)

- [ ] At breakpoint `375px` your mobile version behaves as expected. But from breakpoint `375px` to `768px` white space seems to appear. This is not the expected behaviour of the mobile version. Your app should be responsive. Kindly look into that and fix.

### You app behaviour
[mobile-behaviour.webm](https://user-images.githubusercontent.com/26597432/221517833-7f8de78f-65de-41e9-81f7-86969cb21925.webm)